packages/xstate-react/rollup.config.js

```diff
@@ -0,0 +1,52 @@
+import { terser } from 'rollup-plugin-terser';
+import typescript from 'rollup-plugin-typescript2';
+
+function createTSCofig() {
+  return typescript({
+    tsconfigOverride: {
+      compilerOptions: {
+        declaration: false
+      }
+    }
+  });
+}
+
+function createUmdConfig({ input, output: file, name }) {
+  return {
+    input,
+    output: {
+      file,
+      format: 'umd',
+      name,
+      globals: {
+        xstate: 'XState',
+        '@xstate/react': 'XStateReact',
+        '@react/composition-api': 'reactCompositionApi'
+      }
+    },
+    plugins: [createTSCofig(), terser({ include: [/^.+\.min\.js$/] })]
```

Comment:
as mentioned in the other comment - let's just focus on producing minified bundles